tests/Unit/Evolution/Systems/Cce/Test_Equations.cpp

```diff
@@ -0,0 +1,195 @@
+    tags_needed_for_integrand_computation<
+        Tag, TagsCategory::TagsToSwshDifferentiate>,
+    tags_needed_for_integrand_computation<Tag, TagsCategory::SwshDerivatives>,
+    tags_needed_for_integrand_computation<
+        Tag, TagsCategory::IndependentOfBondiIntegration>,
+    tags_needed_for_integrand_computation<Tag, TagsCategory::Temporary>>>;
+
+template <typename bondi_integrand_tag>
+struct python_function_for_bondi_integrand;
+
+template <>
+struct python_function_for_bondi_integrand<Tags::Integrand<Tags::Beta>> {
+  static std::string name() noexcept { return "compute_integrand_for_beta"; }
+};
+template <>
+struct python_function_for_bondi_integrand<Tags::PoleOfIntegrand<Tags::Q>> {
+  static std::string name() noexcept {
+    return "compute_integrand_for_q_pole_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<Tags::RegularIntegrand<Tags::Q>> {
+  static std::string name() noexcept {
+    return "compute_integrand_for_q_regular_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<Tags::Integrand<Tags::U>> {
+  static std::string name() noexcept { return "compute_integrand_for_u"; }
+};
+template <>
+struct python_function_for_bondi_integrand<Tags::PoleOfIntegrand<Tags::W>> {
+  static std::string name() noexcept {
+    return "compute_integrand_for_w_pole_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<Tags::RegularIntegrand<Tags::W>> {
+  static std::string name() noexcept {
+    return "compute_integrand_for_w_regular_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<Tags::PoleOfIntegrand<Tags::H>> {
+  static std::string name() noexcept {
+    return "compute_integrand_for_h_pole_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<Tags::RegularIntegrand<Tags::H>> {
+  static std::string name() noexcept {
+    return "compute_integrand_for_h_regular_part";
+  }
+};
+template <>
+struct python_function_for_bondi_integrand<Tags::LinearFactor<Tags::H>> {
+  static std::string name() noexcept { return "compute_linear_operator_for_h"; }
+};
+template <>
+struct python_function_for_bondi_integrand<
+    Tags::LinearFactorForConjugate<Tags::H>> {
+  static std::string name() noexcept {
+    return "compute_linear_operator_for_conjugate_h";
+  }
+};
+
+SPECTRE_TEST_CASE("Unit.Evolution.Systems.Cce.Equations", "[Unit][Evolution]") {
+  pypp::SetupLocalPythonEnvironment local_python_env{"Evolution/Systems/Cce/"};
+
+  using all_bondi_tags =
+      tmpl::list<Tags::Beta, Tags::Q, Tags::U, Tags::W, Tags::H>;
+  tmpl::for_each<all_bondi_tags>([](auto x) {
+    using bondi_tag = typename decltype(x)::type;
+    tmpl::for_each<integrand_terms_to_compute_for_bondi_variable<bondi_tag>>(
+        [](auto y) {
+          using bondi_integrand_tag = typename decltype(y)::type;
+          using bondi_integrand_argument_list =
+              all_arguments_for_integrand<bondi_integrand_tag>;
+          // check the Variables interface
```

Comment:
Once the Variables interface is in a test helpers, this comment should give a few more details as to why we are testing this way (i.e. to allow the test to use the equations without using a databox and without manually specifying each of the functions and each of the numerous arguments for each of those functions).